### DISCUSSION TITLE
how can exit from fzf with ctrl+c or/and ESC and not run nano

### DISCUSSION THREAD

**Discussion by og900aero** [junegunn/fzf#3737] [Q&A]
I want to open documents with nano, except when I press ctrl+c or/and ESC.
I try this, but always open nano:
fzf -e --walker-root / --walker-skip proc,tmp,run --preview 'batcat --color=always {}' --preview-window '~3' --print0 | xargs -0 -o nano

I try this method, but doesnt open any document:
fzf -e --walker-root / --walker-skip proc,tmp,run --preview 'batcat --color=always {}' --preview-window '~3' --print0 | test $? -eq 0 && xargs -0 -o nano

**Comment by og900aero** [comment]
result=$(fzf --exact --walker-root / --walker-skip proc,tmp,run --preview 'batcat --color=always {}'); test -n "$result" && { test -w "$result" && nano "$result" || sudo nano "$result"; }

**Comment by junegunn** [comment]
See https://github.com/junegunn/fzf?tab=readme-ov-file#turning-into-a-different-process
